packages/eslint-plugin-pf-codemods/src/rules/v6/expandableSectionRemoveIsActiveProp/expandableSectionRemoveIsActivePropOutput.tsx

```diff
@@ -0,0 +1,5 @@
+import { ExpandableSection } from "@patternfly/react-core";
+
+export const ExpandableSectionRemoveIsActivePropInput = () => (
+  <ExpandableSection isActive />
```

Comment:
This file should have the prop removed and match the expected output (additional white space if there is one).

Reply:
done, ty